frontend/src/App.tsx

```diff
@@ -1,23 +1,26 @@
+import { Routes, Route } from 'react-router-dom'
+import './index.css'
+import { Contacts } from './components/contacts'
+import { ProtectedRoutes } from './components/protectedRoutes'
+import { GetUserContext } from './components/UserContext'
 import { LoginForm } from './components/loginForm'
 import { SignUp } from './components/signup'
 import { TempLandingPage } from './components/tempLandingPage'
-import './index.css'
-import { Routes, Route } from 'react-router-dom'
-import { Contacts } from './components/contacts'
-import { PlanForm } from './components/PlanForm'
-import React from 'react'
 
 function App (): JSX.Element {
   return (
     <div>
-      <Routes>
-        <Route path="/login" element={<LoginForm />} />
-        <Route path="/signup" element={<SignUp />} />
-        <Route path="/tempLandingPage" element={<TempLandingPage />} />
-        <Route path="/contactPage" element={<Contacts />} />
-        <Route path="/planForm" element={<PlanForm />} />
-      </Routes>
-    </div>
+      <GetUserContext>
+        <Routes>
+          <Route element={<ProtectedRoutes />}>
+            <Route path="/contactPage" element={<Contacts />} />
+          </Route>
+          <Route path="/login" element={<LoginForm />} />
+          <Route path="/signup" element={<SignUp />} />
+          <Route path="/tempLandingPage" element={<TempLandingPage />} />
```

Comment:
did you intentionally remove the /planForm page?

Reply:
Added back